dist/immstruct.js

```diff
@@ -653,13 +824,12 @@ function Structure (options) {
     this._currentRevision = 0;
   }
 
-  this._pathListeners = [];
+  this._pathListeners = {};
```

Comment:
How do this work with prefix-checks and the possibility of doing suffix-trees/more advanced lookup for functions?